serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-openapi-based-services.adoc

```diff
@@ -0,0 +1,358 @@
+Orchestration of OpenAPI based services
+=======================================
+:compat-mode!:
+// Metadata:
+:description: Orchestration of OpenAPI based services
+:keywords: kogito, workflow, serverless, openapi, services, json
+// links:
+:swagger_inspector_url: https://swagger.io/docs/swagger-inspector/how-to-create-an-openapi-definition-using-swagger/
+:quarkus_swagger_url: https://quarkus.io/guides/openapi-swaggerui
+:spring_swagger_url: https://springdoc.org/#Introduction
+:go_swagger_url: https://goswagger.io/
+:python_flasgger_url: https://github.com/flasgger/flasgger
+:dotnet_swagger_url: https://docs.microsoft.com/en-us/aspnet/core/tutorials/web-api-help-pages-using-swagger?view=aspnetcore-6.0
+:npm_swagger_url: https://www.npmjs.com/package/swagger-ui-express
+:php_swagger_url: https://github.com/zircote/swagger-php
+:quarkus_config_url: https://quarkus.io/guides/config-reference
+:quarkus_rest_client_url: https://quarkus.io/guides/rest-client
+:mp_config_env_vars_url: https://github.com/eclipse/microprofile-config/blob/master/spec/src/main/asciidoc/configsources.asciidoc#environment-variables-mapping-rules
+// Referenced documentation pages.
+:getting-familiar-with-our-tooling: xref:getting-started/getting-familiar-with-our-tooling.adoc
+:create-your-first-workflow-service: xref:getting-started/create-your-first-workflow-service.adoc
+:build-workflow-image-with-quarkus-cli: xref:cloud/build-workflow-image-with-quarkus-cli.adoc
+:understanding-jq-expressions: xref:core/understanding-jq-expressions.adoc
+:configuring-openapi-services-endpoints: xref:service-orchestration/configuring-openapi-services-endpoints.adoc
+:camel-k-integration: xref:integrations/camel-k-integration.adoc
+:authention-support-for-openapi-services: xref:security/authention-support-for-openapi-services.adoc
+
+This document describes how to call REST services via an link:{open_api_spec_url}[OpenAPI] specification file.
+
+The examples in this document assume that you have the Quarkus tooling installed and have created a Serverless Workflow project. 
+
+For more information about the tooling, see {getting-familiar-with-our-tooling}[Using Serverless Workflow tooling]. To create a new project, see {create-your-first-workflow-service}[Creating your first Serverless Workflow service].
+
+== The OpenAPI Function Definition
+
+Kogito follows the link:{spec_doc_url}#using-functions-for-restful-service-invocations[CNCF Serverless Workflow specification function definition] to make calls to remote REST services using an OpenAPI specification reference, as shown in the following example:
+
+.An example of an OpenAPI function definition
```

Comment:
I think we should mention that type of the function doesn't have to be defined here because it's implicitly set to "rest".